pkg/csource/csource_test.go

```diff
@@ -238,6 +238,17 @@ syscall(SYS_csource7, /*flag=BIT_1*/2ul);
 syscall(SYS_csource7, /*flag=BIT_0_AND_1*/3ul);
 syscall(SYS_csource7, /*flag=*/4ul);
 syscall(SYS_csource7, /*flag=BIT_0|0x4*/5ul);
+`,
+		},
+
+		{
+			input: `
+csource0(0xffffffff)
+csource8(0xffffffffffffffff)
+`,
+			output: `
+syscall(SYS_csource0, /*num=*/(intptr_t)-1);
+syscall(SYS_csource8, /*num=*/(intptr_t)-1);
```

Comment:
What will happen for 32-bit arch? intptr_t is 32 bit there.

Reply:
Added a 32 bit test as well. I guess `(intptr_t)-1` is still reasonable?

Reply:
> UPD The syscall(SYS_csource8, /*num=*/(intptr_t)-1); case for csource8(num int64) seems weird though

Right. IIRC there 1 such case. But that's a kernel bug. It can't work.